src/platform_impl/windows/definitions.rs

```diff
@@ -3,11 +3,15 @@
 
 use std::ffi::c_void;
 
-use windows_sys::core::{IUnknown, GUID, HRESULT};
+use windows_sys::core::{GUID, HRESULT};
 use windows_sys::Win32::Foundation::{BOOL, HWND, POINTL};
-use windows_sys::Win32::System::Com::{
-    IAdviseSink, IDataObject, IEnumFORMATETC, IEnumSTATDATA, FORMATETC, STGMEDIUM,
-};
+use windows_sys::Win32::System::Com::{FORMATETC, STGMEDIUM};
+
+pub type IUnknown = *mut c_void;
+pub type IAdviseSink = *mut c_void;
+pub type IDataObject = *mut c_void;
+pub type IEnumFORMATETC = *mut c_void;
+pub type IEnumSTATDATA = *mut c_void;
```

Comment:
These are available in `windows-sys`? Why not just use the types from there?

Reply:
They were removed from v0.59. Not sure why; it could just be because they're just aliases to `*mut c_void`. You have to pull in `windows-core` or `windows` to get those types now.

Reply:
Fair enough, thanks for the answer.